openidm-zip/src/main/resources/db/h2/scripts/openidm.sql

```diff
@@ -0,0 +1,482 @@
+
+CREATE SCHEMA IF NOT EXISTS openidm AUTHORIZATION openidm;
+
+-- -----------------------------------------------------
+-- Table openidm.objecttpyes
+-- -----------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS openidm.objecttypes (
+  id BIGSERIAL NOT NULL,
+  objecttype VARCHAR(255) DEFAULT NULL,
+  PRIMARY KEY (id),
+  CONSTRAINT idx_objecttypes_objecttype UNIQUE (objecttype)
+);
+
+-- -----------------------------------------------------
+-- Table openidm.genericobjects
+-- -----------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS openidm.genericobjects (
+  id BIGSERIAL NOT NULL,
+  objecttypes_id BIGINT NOT NULL,
+  objectid VARCHAR(255) NOT NULL,
+  rev VARCHAR(38) NOT NULL,
+  fullobject JSON,
+  PRIMARY KEY (id),
+  CONSTRAINT fk_genericobjects_objecttypes FOREIGN KEY (objecttypes_id) REFERENCES openidm.objecttypes (id) ON DELETE CASCADE ON UPDATE NO ACTION,
+  CONSTRAINT idx_genericobjects_object UNIQUE (objecttypes_id, objectid)
+);
+
+-- -----------------------------------------------------
+-- Table openidm.genericobjectproperties
+-- -----------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS openidm.genericobjectproperties (
+  genericobjects_id BIGINT NOT NULL,
+  propkey VARCHAR(255) NOT NULL,
+  proptype VARCHAR(32) DEFAULT NULL,
+  propvalue TEXT,
+  CONSTRAINT fk_genericobjectproperties_genericobjects FOREIGN KEY (genericobjects_id) REFERENCES openidm.genericobjects (id) ON DELETE CASCADE ON UPDATE NO ACTION
+);
+CREATE INDEX IF NOT EXISTS fk_genericobjectproperties_genericobjects ON openidm.genericobjectproperties (genericobjects_id);
+--H2 cannot create indexes on clobs yet (1.4.200)
+-- CREATE INDEX idx_genericobjectproperties_prop ON openidm.genericobjectproperties (propkey,propvalue);
+
+-- -----------------------------------------------------
+-- Table openidm.managedobjects
+-- -----------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS openidm.managedobjects (
+  id BIGSERIAL NOT NULL,
+  objecttypes_id BIGINT NOT NULL,
+  objectid VARCHAR(255) NOT NULL,
+  rev VARCHAR(38) NOT NULL,
+  fullobject JSON,
+  PRIMARY KEY (id),
+  CONSTRAINT fk_managedobjects_objectypes FOREIGN KEY (objecttypes_id) REFERENCES openidm.objecttypes (id) ON DELETE CASCADE ON UPDATE NO ACTION
+);
+
+CREATE UNIQUE INDEX IF NOT EXISTS idx_managedobjects_object ON openidm.managedobjects (objecttypes_id,objectid);
+-- Note that the next two indices apply only to role objects, as only role objects have a condition or temporalConstraints
+-- CREATE INDEX IF NOT EXISTS idx_json_managedobjects_roleCondition ON openidm.managedobjects
```

Comment:
Why leave those comments here?

Reply:
Index removed. `json_extract_path*` is postgres-specific. 
This index doesn't seem to be crucial, because current non-postgres scripts don't create it.